packages/core/admin/admin/src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export function useDebounce<Value>(value: Value, delay: number): Value {
```

Comment:
```suggestion
export function useDebounce<TValue>(value: TValue, delay: number): TValue {
```

iirc we're appending generics with T to not be confused with types 👍🏼